membership_policy.md

```diff
@@ -239,6 +245,24 @@ After an extended period away from the project with no activity
 those members would need to re-familiarize themselves with the current state
 before being able to contribute effectively.
 
+### Inactive Reviewers/Approvers/Chairs/Leads
+
+Reviewers, approvers, chairs and leads are pillars of the community and are fundamental to maintaining the momentum of the project.
+Considering that they have elevated permissions and responsibilities in the project, their active engagement with the project is greatly appreciated.
+
+If a reviewer and/or approver steps down from their position in a repository or the project entirely, we trust that they will notify the community and create PRs to remove themselves as reviewers/approvers in the applicable repositories.
+This will help others identify and grow into the role so as to not leave a gap for any length of time. 
+It will also ensure they are not erroneously tagged to review/approve PRs, which is both an inconvenience to them but also an obstacle for the project.
+
+When a SIG or WG Chair or Subproject Lead needs to step down from their position, it is expected that they work with the SIG/Subproject/WG to identify and mentor a willing candidate to grow into the role.
+They should then open a PR to replace them in [sigs.yaml], and move themselves to emeritus status (if applicable).
+
+A reviewer/approver/chair/lead may also be demoted or removed from their role if they are inactive in their area for 6 months or more, or if there is a majority vote by the remaining approvers/chairs/leads or maintainers depending on the situation (for instance, if a SIG chair abruptly leaves one remaining SIG chair).
+
+#### How inactivity for Reviewers/Approvers/Chairs/Leads is measured
+
+The KubeVirt community repo has a [project activity tool](https://github.com/kubevirt/community/blob/main/generators/cmd/contributions/README.md) that can be used to demonstrate that a reviewer/approver has been inactive for 6 months.
```

Comment:
@dhiller don't we have a periodic execution of this, with public-facing output? I think a link to such output would be more valuable than a link to the readme of the tool.

Reply:
> @dhiller don't we have a periodic execution of this, with public-facing output? I think a link to such output would be more valuable than a link to the readme of the tool.

No, we don't. My first question would be how we would want to present this though.

I was trying to tackle this in a different way: by automating the removal of accounts who don't show activity - this way the process is democratized. However the PR didn't get much feedback, not sure why that is.

https://github.com/kubevirt/project-infra/pull/3832

Reply:
^^ @jean-edouard